ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -267,7 +267,7 @@ message GrantUserRightsRequest {
     string user_id = 1;
 
     // The rights to grant.
-    // Required
+    // Optional
```

Comment:
Is that really optional? What's the use of a grant rights call where this field is not populated?

Reply:
It's how the implementation works. We never enforced the rights in the request to be non-empty.
We could change it, but then it's a breaking change in the LAPI

Reply:
ok